modules/host/ghaf_host_hardened_baseline

```diff
@@ -812,28 +864,83 @@ CONFIG_UNIX=y
 CONFIG_UNIX_SCM=y
 CONFIG_AF_UNIX_OOB=y
 # CONFIG_UNIX_DIAG is not set
-# CONFIG_INET is not set
```

Comment:
We could probably put all these networking 
https://github.com/tiiuae/ghaf/pull/415/files#diff-bc4e2d3616e697e47c5c01ea825ca521e9c6ad6ce46288131c064c18304ced97L815-R985
related enabling/hardening options also behind a ghaf option and `structuredExtraConfig`.

The rationale for that is that we could test the baseline hardenened kernel config without networking instead of enabling and hardening the network to comply with `kernel-hardening-checker` checks. 
Ghaf hardened host will require local hardened networking, maybe with IPv6 supported options, but having an option to remove host networking at platform baseline hardening level is worth the future effort.

We can scope that work out of this PR.

Reply:
Sure @vilvo